lib/mrsk/commands/accessory.rb

```diff
@@ -57,12 +58,14 @@ def execute_in_existing_container(*command, interactive: false)
       *command
   end
 
-  def execute_in_new_container(*command, interactive: false)
+  def execute_in_new_container(*command, interactive: false, labels: [])
     docker :run,
       ("-it" if interactive),
       "--rm",
       *env_args,
       *volume_args,
+      "--label", "custom=true",
```

Comment:
I think we should also add `config.label_args` here so the custom containers are labeled with the service and destination. 

We'll need to wait for https://github.com/mrsked/mrsk/pull/183 to land first though so we can provide a clear upgrade path.

We could also consider giving the container a meaningful name, maybe using `custom` as a special role and appending a random string to allow multiple instances.